roles/hosted_engine_setup/hooks/after_setup/add_host_storage_domain.yml

```diff
@@ -1,15 +1,15 @@
 ---
 - name: Include Host vars
-  include_vars: "{{ file_item }}"
+  ansible.builtin.include_vars: "{{ file_item }}"
   with_fileglob: "/usr/share/ovirt-hosted-engine-setup/gdeploy-inventory.yml"
   loop_control:
     loop_var: file_item
-- debug: var=gluster
+- ansible.builtin.debug: var=gluster
 
 - name: Set Engine public key as authorized key without validating the TLS/SSL certificates
   connection: ssh
   authorized_key:
-    user: root
+    ansible.builtin.user: root
```

Comment:
It's a `user` parameter and not a call to `ansible.builtin.user` module, right?

Reply:
Yes. I wonder if there is a simple way to find out only user module calls. Will check/fix manually for now.